docs/rfcs/2023-09-12-remote-profile.md

```diff
@@ -46,15 +46,20 @@ This can be solved in a few different ways:
 3. The mobile may start announcing itself with UDP broadcasts for the duration of the phase (bluetooth-style) using information in the QR code.
 4. A desktop may create a temporary SMP queue and show its address. The mobile then submits its server data to it.
 
+Another option is to run the server on desktop and have mobile discover it with the help of QR code to get server identity and keys and then on the network via some protocol. Using a fixed address is suboptimal as most networks have dynamic IPs.
+
 ### Handshake
 
 The aim of this phase is to establish a TLS+cryptobox session.
 
+TLS could be complex as we need to generate self-signed certificates on desktop (if it acts like a server). A plaintext ws connection with cryptobox encryption could be sufficient initially? 
+
```

Comment:
Haskell-side, Ed25519 looks trivial to generate and pack into X509 cert. Post a fingerprint and you have it.